docs/reference/security/fips-140-compliance.asciidoc

```diff
@@ -8,59 +8,75 @@ government computer security standard used to approve cryptographic modules.
 {es} offers a FIPS 140-2 compliant mode and as such can run in a FIPS 140-2
 configured JVM.
 
-IMPORTANT: The JVM bundled with {es} is not configured for FIPS 140-2. You must 
-configure an external JDK with a FIPS 140-2 certified Java Security Provider.
+IMPORTANT: The JVM bundled with {es} is not configured for FIPS 140-2. You must
+configure an external JDK and it highly recommended to use a FIPS 140-2 certified Java Security Provider.
```

Comment:
I'm not sure about this change, could you please clarify more? If you don't use a certified provider, the whole setup will not be compliant.